contrib/unit_test_coverage/generate_coverage_report.sh

```diff
@@ -77,9 +75,8 @@ if [ ! -d $OUTPUT_DIR ]; then
   exit 1
 fi
 
-LIBS=$(ls $BUILD_DIR/lib/* | grep -v "$LIB_FILE_FILTER")
 BINS=$(find $BUILD_DIR/bin/ -maxdepth 1 -type f | grep -v "$BIN_FILE_FILTER")
-FORMATTED_BINARY_FILES=$(echo $LIBS $BINS | sed "s/ / -object=/g")
```

Comment:
With LLVM-11, llvm-cov doesn't accept static libraries anymore. Not sure if that was ever desired. Only offering the test binaries seem to yield the same results. (I checked the generated report against a previous report and the numbers are similar, not exactly the same though.)

Reply:
The numbers not being 100% percent the same, might also be due to non-deterministic tests. I think its fine